RotaryInvertedPendulum-arduino/PIDControl/PIDControl.ino

```diff
@@ -59,12 +59,110 @@ State state = WAITING;
 int counterPlot = 0;
 int frequencyPlot = 20;
```

Comment:
These can be deleted because the print method is now checking `if (t1 - t0 > 10)`, right?